plugins/analytics-module-new-relic/README.md

```diff
@@ -0,0 +1,103 @@
+# Analytics Module: New Relic Browser
+
+This plugin provides an opinionated implementation of the Backstage Analytics API for New Relic Browser. Once installed and configured, analytics events will be sent to New Relic as your users navigate and use your Backstage instance.
+
+This plugin contains no other functionality.
+
+## Installation
+
+1. Install the plugin package in your Backstage app:
+
+```sh
+# From your Backstage root directory
+yarn add --cwd packages/app @backstage/plugin-analytics-module-nr
+```
+
+2. Wire up the API implementation to your App:
+
+```tsx
+// packages/app/src/apis.ts
+import {
+  analyticsApiRef,
+  configApiRef,
+  identityApiRef,
+} from '@backstage/core-plugin-api';
+import { NewRelicBrowser } from '@backstage/plugin-analytics-module-nr';
+
+export const apis: AnyApiFactory[] = [
+  // Instantiate and register the New Relic Browser API Implementation.
+  createApiFactory({
+    api: analyticsApiRef,
+    deps: { configApi: configApiRef, identityApi: identityApiRef },
+    factory: ({ configApi, identityApi }) =>
+      NewRelicBrowser.fromConfig(configApi, {
+        identityApi,
+      }),
```

Comment:
Would it be Worth highlighting the optionality of this? I assume the reason you made it possible to omit is for privacy reasons.

Reply:
Good point. I've added a section about User ID's to the README. I noticed that the Google Analytics implementation also calculates a hash of the user entity reference by default. Perhaps we should add that here as well?

Reply:
The GA implementation was following Google guidance on minimizing PII (a terms of service issue); you could take a similar approach if it makes sense re: NR's terms of service.

Reply:
I couldn't really find any guidance on this in the New Relic docs, so I added a similar implementation as the GA one. I guess folks can always just return the original user ID if they want to, but then at least it is an opt-in rather than default behaviour.